projects/plugins/jetpack/modules/contact-form/admin.php

```diff
@@ -19,9 +20,12 @@
 /**
  * Build contact form button.
  *
+ * @deprecated $$next-version$$ Use Automattic\Jetpack\Forms\ContactForm\
  * @return void
  */
 function grunion_media_button() {
+	_deprecated_function( __FUNCTION__, '$$next-version$$', 'Automattic\Jetpack\Forms\ContactForm\Admin->' );
```

Comment:
I think we should add `jetpack-` to all those deprecation notices, as recommended here:
https://github.com/Automattic/jetpack/blob/f0fa2c82dce0cb507b892645e27a9fafd8ab87e0/docs/coding-guidelines.md#L71

This is helpful in error logs, folks know the notices are about the Jetpack plugin, and not about core.

```suggestion
	_deprecated_function( __FUNCTION__, 'jetpack-$$next-version$$', 'Automattic\Jetpack\Forms\ContactForm\Admin->grunion_media_button' );
```

Reply:
Good catch!

Reply:
Thanks for the tip!